src/components/App/index.tsx

```diff
@@ -9,6 +9,9 @@ import '../../lib/SnackBar';
 import Intro from '../intro';
 import '../custom-els/LoadingSpinner';
 
+// This is imported for TypeScript only. It isn't used.
+import Compress from '../compress';
```

Comment:
Use the `import('../compress')` syntax instead then?

Reply:
Oh, I think I wrote this, but it's been replaced. Maybe it crept back in in a rebase?